PowerEditor/src/NppDarkMode.h

```diff
@@ -153,6 +185,7 @@ namespace NppDarkMode
 
 	// handle events
 	void handleSettingChange(HWND hwnd, LPARAM lParam);
```

Comment:
change to `void handleSettingChange(HWND hwnd, LPARAM lParam, bool isFromBtn = false)`